package.json

```diff
@@ -46,6 +46,7 @@
     "instrument": true
   },
   "dependencies": {
+    "arrify": "^2.0.1",
```

Comment:
What is this?

Reply:
In the state of the world (or my laptop?) a few weeks ago, it seemed necessary to pin this subdependency to this version number to prevent compilation errors with this version of typescript. Can't reproduce it now, assuming it got solved, so removing this again. It's not a direct dependency after all.